dashboard/modules/node/tests/test_node.py

```diff
@@ -526,5 +528,33 @@ def check_logs():
     )
 
 
+# See detail: https://github.com/ray-project/ray/issues/24361
+@pytest.mark.skipif(sys.platform == "win32", reason="Flaky on Windows.")
```

Comment:
is this window relevant?

Reply:
No, but this case relies on `os.getpid` to get the agent PID and check port number, but according to this: https://github.com/python/cpython/issues/83086#issuecomment-1129708440, `os.getpid` seems not reliable on windows